CI/Import/Variables.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 TMP_DIR="/tmp"
 
+#CI_RESULTS_REPO="https://github.com/ILIAS-eLearning/CI-Results"
+CI_RESULTS_REPO="https://github.com/lauraquellmalz/CI-Results"
+
 PHP_CS_FIXER_RESULTS_PATH="$TMP_DIR/phpfix_results"
 PHP_CS_FIXER="libs/composer/vendor/friendsofphp/php-cs-fixer/php-cs-fixer"
 
-PHP_VERSION=$(phpenv version-name)
+PHP_VERSION=$(php -v | tac | tail -n 1 | cut -d " " -f 2 | cut -c 1-3)
```

Comment:
Don't wonder please, I couldn't find an easier way to get the php version. This one is based on pure bash tools without any other software like phpenv in between. So we can move this script in the future somewhere else if needed.